From 258c18646ebf0ba92c7f3c1f601fdf718790e419 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Thu, 12 May 2011 15:40:06 +0800
Subject: [PATCH] Revert "RK29MobileSDK: Open Uart0 CLK when system in sleep;"

This reverts commit 9e3e1986b475e5d434f99669c3fc94751138665c.
Only revert arch/arm/mach-rk29/pm.c
---
 arch/arm/mach-rk29/pm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-rk29/pm.c b/arch/arm/mach-rk29/pm.c
index 044952c3458e..3a02514ef884 100755
--- a/arch/arm/mach-rk29/pm.c
+++ b/arch/arm/mach-rk29/pm.c
@@ -404,7 +404,7 @@ static int rk29_pm_enter(suspend_state_t state)
 	clkgate[1] = cru_readl(CRU_CLKGATE1_CON);
 	clkgate[2] = cru_readl(CRU_CLKGATE2_CON);
 	clkgate[3] = cru_clkgate3_con_mirror;
-	cru_writel((~((1 << CLK_GATE_CORE)
+	cru_writel(~((1 << CLK_GATE_CORE)
 		   | (1 << CLK_GATE_ACLK_CPU)
 		   | (1 << CLK_GATE_ACLK_CPU2)
 		   | (1 << CLK_GATE_PCLK_CPU)
@@ -416,7 +416,7 @@ static int rk29_pm_enter(suspend_state_t state)
 		   | (1 << CLK_GATE_GPIO0)
 		   | (1 << CLK_GATE_RTC)
 		   | (1 << CLK_GATE_GRF)
-		   ) | clkgate[0])&(~(1 << CLK_GATE_UART0)), CRU_CLKGATE0_CON);
+		   ) | clkgate[0], CRU_CLKGATE0_CON);
 	cru_writel(~0, CRU_CLKGATE1_CON);
 	cru_writel(~((1 << CLK_GATE_GPIO1 % 32)
 		   | (1 << CLK_GATE_GPIO2 % 32)
-- 
2.35.3

